.github/workflows/integration.yml

```diff
@@ -271,6 +271,26 @@ jobs:
       - name: build proposals tests
         run: yarn build
         working-directory: a3p-integration
+      - name: zip up all core proposals
+        run: |
+          find . -type d -path "./a3p-integration/proposals/*/submission" | while read -r dir; do
+            parent_dir=$(basename "$(dirname "$dir")")
+
+            sanitized_parent_dir=${parent_dir//:/-}
+
+            zip_file="${sanitized_parent_dir}.zip"
+
+            zip -r "$zip_file" "$dir"
+
+            echo "Zipped $dir into $zip_file"
+          done
+      - name: archive core proposals
+        uses: actions/upload-artifact@v4
+        with:
+          name: core-proposals
+          path: ./*.zip
+      - name: remove all zip files after they're archived
```

Comment:
This is required to pass the git tree dirty check later on in this workflow

Reply:
could you create the zips in a path outside the repo? e.g. `/tmp/proposal-zips`

Reply:
or in the repo but under a .gitignored path. e.g. `build`. That should avoid the dirty check.

We shouldn't need another step for cleanup